lib/bowling_score.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+# BowlingScore used to take a single string showing the score of each throw of
+# the game as input and calculate the total score
+class BowlingScore
+  attr_accessor :all_throws
+
+  def initialize(all_throws, full_game = false)
+    @all_throws = all_throws.split('')
```

Comment:
I agree this would be a better way to do things, but the kata had rules which meant I couldn't:

- We will not check for valid rolls.
- We will not check for correct number of rolls and frames.
- We will not provide scores for intermediate frames.

I could add this functionality on a new feature branch though!